### ISSUE TITLE
Package error: module `src` not found

### ISSUE THREAD

**Issue by adwait** [dobios/btor2-opt#5]
`import btoropt` fails with the following error:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "btoropt/__init__.py", line 21, in <module>
    from .passes.genericpass import *
  File "btoropt/passes/genericpass.py", line 21, in <module>
    from src.btoropt.program import Instruction
ModuleNotFoundError: No module named 'src'
```

I think this should be made relative:

https://github.com/dobios/btor2-opt/blob/3703885db0e32dd501d221c824312f652f476d95/src/btoropt/passes/genericpass.py#L21

**Comment by dobios** [comment]
Damn this worked just fine installed locally and works just fine in the CI. That's weird that it doesn't work when packaged. I'll make everything relative then.

Out of curiosity, what OS are you on @adwait ? Both my local machine and the CI are on Linux. Might be a Windows vs Unix build be slightly different issue as usual lol

**Comment by dobios** [comment]
Fixed in #6 , updated pypi package
